values.yaml

```diff
@@ -485,6 +485,18 @@ datawolf:
     postgresqlPassword: datawolf
     postgresqlDatabase: database
 
+  extraEnvVars:
+    - name: DATASET_DAO
+      value: "edu.illinois.ncsa.incore.dao.IncoreDatasetDao"
```

Comment:
Will this cause any problem for prod? For the community apps, we don't want to set the dataset day to the Incore one.

Reply:
This shouldn't matter since this is only for the extra env vars. As long as we don't update the datawolf container image, it shouldn't affect it. It will just simply be an extra environment variable that is not being used.

Reply:
@navarroc Can you approve this?